### ISSUE TITLE
0.8.1 broken on Windows

### ISSUE THREAD

**Issue by zhanghaocars** [robocorp/robotframework-lsp#283]
**Describe the bug**
0.8.1 broken on Windows

**To Reproduce**
1. In "RobotFramework" Output:
```
Getting language server Python executable.
Language server Python executable gotten from robocorp.getLanguageServerPython.
Plugins dir: c:\Users\Administrator\.vscode\extensions\robocorp.robocorp-code-0.8.1\src\robocorp_code\plugins.
Starting RobotFramework Language Server with args: C:\Users\Administrator\AppData\Local\robocorp\live\2aa1db4bb1a50e5e\python.exe,-u,c:\Users\Administrator\.vscode\extensions\robocorp.robotframework-lsp-0.11.0\src\robotframework_ls\__main__.py
Waiting for RobotFramework (python) Language Server to finish activating...
lsp: 2021-03-25 18:32:53 UTC pid: 6012 - MainThread - EXCEPTION - robotframework_ls.server_manager
Error starting robotframework server api (server_process=None).

Traceback (most recent call last):
  File "c:\Users\Administrator\.vscode\extensions\robocorp.robotframework-lsp-0.11.0\src\robotframework_ls\server_manager.py", line 252, in get_robotframework_api_client
    server_process = start_server_process(
  File "c:\Users\Administrator\.vscode\extensions\robocorp.robotframework-lsp-0.11.0\src\robotframework_ls\server_api\server__main__.py", line 71, in start_server_process
    language_server_process = subprocess.Popen(
  File "C:\Users\Administrator\AppData\Local\robocorp\live\2aa1db4bb1a50e5e\lib\subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\Administrator\AppData\Local\robocorp\live\2aa1db4bb1a50e5e\lib\subprocess.py", line 1311, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
ValueError: illegal environment variable name

RobotFramework Language Server ready. Took: 5.7s
lsp: 2021-03-25 18:32:53 UTC pid: 6012 - MainThread - EXCEPTION - robotframework_ls.server_manager
Error starting robotframework server api (server_process=None).

Traceback (most recent call last):
  File "c:\Users\Administrator\.vscode\extensions\robocorp.robotframework-lsp-0.11.0\src\robotframework_ls\server_manager.py", line 252, in get_robotframework_api_client
    server_process = start_server_process(
  File "c:\Users\Administrator\.vscode\extensions\robocorp.robotframework-lsp-0.11.0\src\robotframework_ls\server_api\server__main__.py", line 71, in start_server_process
    language_server_process = subprocess.Popen(
  File "C:\Users\Administrator\AppData\Local\robocorp\live\2aa1db4bb1a50e5e\lib\subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\Administrator\AppData\Local\robocorp\live\2aa1db4bb1a50e5e\lib\subprocess.py", line 1311, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
ValueError: illegal environment variable name
```
2. `0.8.0` do not have this problem.

**Versions:**
 -  [Windows 10]
 - Robot Framework Version [ 4.0]
 - Robot Framework Language Server Version [ 0.8.0]
 - Client Version [e.g.: VSCode 1.54.3]

**Comment by fabioz** [comment]
To debug this, can you do the following:

1. Set `LSP_DEBUG_PROCESS_ENVIRON=1` in your environment variables
2. Provide the logs (as explained in: https://github.com/robocorp/robotframework-lsp/blob/master/robotframework-ls/docs/reporting_issues.md).

**Comment by zhanghaocars** [comment]
@fabioz This issue is easy to reproduce, and I reproduced it in a fresh installed computer.

**Comment by fabioz** [comment]
Ok, I'm going to close it then... if you can reproduce, please reopen.

As a note, from what I investigated, Python raises this if an environment variable with a value that contains `=` is created.

**Comment by fabioz** [comment]
I was actually able to reproduce it here, so, I'll provide a fix shortly and release 0.8.2.
